docs/components_page/components/button_group/radios.R

```diff
@@ -6,8 +6,9 @@ button_group <- htmlDiv(
     dbcRadioItems(
       id = "radios",
       class_name = "btn-group",
-      label_class_name = "btn btn-secondary",
-      label_checked_class_name = "active",
+      input_class_name="btn-check",
+      label_class_name = "btn btn-outline-primary",
+      label_checked_class_name = "active btn-primary",
```

Comment:
`btn-primary` isn't needed here, just `active`

```suggestion
      label_checked_class_name = "active",
```